draft-ietf-oauth-identity-chaining.md

```diff
@@ -133,6 +135,28 @@ The flow illustrated in Figure 1 shows the steps the client in trust Domain A ne
 ## Authorization Server Discovery
 This specification does not define authorization server discovery. A client MAY maintain a static mapping or use other means to identify the authorization server. The `authorization_servers` property in {{I-D.ietf-oauth-resource-metadata}} MAY be used.
 
+## Sender Constrained Tokens
+Either authorization server MAY issue sender constrained tokens. There are currently two options for sender constrained tokens: Mutually Authenticated TLS (mTLS) {{RFC8705}} and Demonstrating Proof of Possessions (DPoP) {{RFC9449}}.
+
+This document introduces the following authorization server metadata values to indicate the authorization server's support for DPoP or mTLS sender constrained tokens.
```

Comment:
I think "authorization server's support for DPoP or mTLS sender constrained tokens" is much too general

Reply:
similar thoughts about below text too